docs/build/html/_sources/usage.rst.txt

```diff
@@ -0,0 +1,34 @@
+Usage
```

Comment:
Doesn't seem like this file belongs here

Reply:
Thanks. This was left here from a readthedocs tutorial template. Removed.